R/zzz.R

```diff
@@ -0,0 +1,6 @@
+#' @importFrom traduire t_
+NULL
+
+.onLoad <- function(...) {
+  hintr:::hintr_init_traduire() # nocov
```

Comment:
At the moment translations are half-included it seems - the dependencies are there but not the translations and not switching in the api? I thought you'd spent some time fighting with that though?

Reply:
Yep left that as needing some work on `traduire` but that has been merged in now. I've added one example of using that - I think that was the only place in this package that we need to use the translator so far but as you say the setup is there if we need more when doing the rest of the refactor.